origami.py

```diff
@@ -1,66 +1,66 @@
-from __future__ import division
-import sublime, sublime_plugin
+import sublime
+import sublime_plugin
 import copy
 from functools import partial
 
-XMIN, YMIN, XMAX, YMAX = list(range(4))
+
+XMIN, YMIN, XMAX, YMAX = range(4)
+
 
 def increment_if_greater_or_equal(x, threshold):
 	if x >= threshold:
-		return x+1
+		return x + 1
 	return x
 
+
 def decrement_if_greater(x, threshold):
 	if x > threshold:
-		return x-1
+		return x - 1
 	return x
 
+
 def pull_up_cells_after(cells, threshold):
-	return [	[x0,decrement_if_greater(y0, threshold),
-				x1,decrement_if_greater(y1, threshold)] for (x0,y0,x1,y1) in cells]
+	return [[x0, decrement_if_greater(y0, threshold),
+			 x1, decrement_if_greater(y1, threshold)] for (x0, y0, x1, y1) in cells]
+
 
 def push_right_cells_after(cells, threshold):
-	return [	[increment_if_greater_or_equal(x0, threshold),y0,
-				increment_if_greater_or_equal(x1, threshold),y1] for (x0,y0,x1,y1) in cells]
+	return [[increment_if_greater_or_equal(x0, threshold), y0,
+			 increment_if_greater_or_equal(x1, threshold), y1] for (x0, y0, x1, y1) in cells]
+
 
 def push_down_cells_after(cells, threshold):
-	return [	[x0,increment_if_greater_or_equal(y0, threshold),
-				x1,increment_if_greater_or_equal(y1, threshold)] for (x0,y0,x1,y1) in cells]
+	return [[x0, increment_if_greater_or_equal(y0, threshold),
+			 x1, increment_if_greater_or_equal(y1, threshold)] for (x0, y0, x1, y1) in cells]
+
 
 def pull_left_cells_after(cells, threshold):
-	return [	[decrement_if_greater(x0, threshold),y0,
-				decrement_if_greater(x1, threshold),y1] for (x0,y0,x1,y1) in cells]
+	return [[decrement_if_greater(x0, threshold), y0,
+			 decrement_if_greater(x1, threshold), y1] for (x0, y0, x1, y1) in cells]
+
 
 def opposite_direction(direction):
-	opposites = {"up":"down", "right":"left", "down":"up", "left":"right"}
-	return opposites[direction]
+	if direction == "up":
+		return "down"
+	if direction == "right":
+		return "left"
+	if direction == "down":
+		return "up"
+	if direction == "left":
+		return "right"
```

Comment:
The previous approach seemed pretty OK to me. Any specific reason for changing that besides personal preference?

Reply:
I'll resolve this since it might be beneficial to write it your way for the sake of code checker (flow analyzer).